packs/slack/config.yaml

```diff
@@ -1,4 +1,26 @@
 ---
+# Used for auth test action
+auth_test_action:
+  token: ""
+
+# Used for list channels
+list_channels_action:
```

Comment:
To me it kinda seems that most of those configuration options (except the token) should really just be action parameters.